ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1007,7 +1007,7 @@ presubmits:
     name: pull-ci-openshift-installer-master-e2e-crc
     optional: true
     rerun_command: /test e2e-crc
-    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(aws|azure|baremetal|kubevirt|libvirt|openstack|ovirt|vsphere)/)
```

Comment:
is crc associated with libvirt? that is, should we be running for changes to libvirt. not sure.

Reply:
wait, based on this pattern it looks like crc is associated with gcp... is that accurate?

Reply:
So, confession time: I have no idea what the -crc job is. The job contains this block:

      - name: cluster-profile
        projected:
          sources:
          - secret:
              name: cluster-secrets-gcp
          - configMap:
              name: cluster-profile-gcp

which suggested GCP to me, but if you know the actual requirements for when this job should [not] run then I'd be happy to modify. Right now it will run on changes to the GCP platform as well as changes to non-platform-specific code.

Reply:
crc is code-ready containers: minishift for 4.x, single-node openshift that runs locally. crc uses libvirt so it would be more appropriate to run this with libvirt changes. the profile/secret is referring to the fact that the job itself runs on top of gcp, but the relevant code being tested is for libvirt.

So can you switch this one to test when changes are for libirt--not gcp? then we should be set.

Reply:
> So can you switch this one to test when changes are for libvirt--not gcp? then we should be set.

Done.